### PR TITLE
Logging - Actions.py - Add

### PR DESCRIPTION
**Pull request by cprafullchandra**

### CODE REVIEW

**Comment by cglewis** [comment]
this seems to have overlapping changes for another PR that isn't merged, please branch from upstream experimental instead.

**Comment by cprafullchandra** [comment]
@cglewis Yeah it has all the components from the previous PR. I'm just putting this here to verify that Travis likes it. Will update commits after previous PR gets merged.

**Comment by cglewis** [comment]
@cprafullchandra it's been merged, please sync up, thanks!

File: vent/api/actions.py [hunk 1]
```diff
@@ -66,13 +73,13 @@ def remove(self, repo=None, namespace=None, name=None, groups=None,
...
 
     def prep_start(self,
-             repo=None,
-             name=None,
-             groups=None,
-             enabled="yes",
-             branch="master",
-             version="HEAD",
-             run_build=False):
+              repo=None,
```
Comment (cglewis) [inline vent/api/actions.py]: if you're going to move these, they should be aligned with `self`

File: vent/api/actions.py [hunk 2]
```diff
@@ -148,7 +155,7 @@ def prep_start(self,
 
             # add extra labels
             if 'labels' not in tool_dict[container_name]:
-                tool_dict[container_name]['labels'] = {}
+               tool_dict[container_name]['labels'] = {}
```
Comment (cglewis) [inline vent/api/actions.py]: shouldn't this be 4 spaces?

File: vent/api/actions.py [hunk 3]
```diff
@@ -365,9 +372,9 @@ def build(self,
         for section in sections:
             self.logger.info("Building "+str(section)+" ...")
             template = self.plugin.builder(template, sections[section]['path'],
-                                        sections[section]['image_name'],
-                                        section, build=True, branch=branch,
-                                        version=version)
+                                         sections[section]['image_name'],
```
Comment (cglewis) [inline vent/api/actions.py]: align with `template`

File: vent/api/actions.py (hunk 1 above)
Comment (cprafullchandra) [inline vent/api/actions.py]: Done!

File: vent/api/actions.py (hunk 2 above)
Comment (cprafullchandra) [inline vent/api/actions.py]: Yep great catch! Fixed.

File: vent/api/actions.py (hunk 3 above)
Comment (cprafullchandra) [inline vent/api/actions.py]: Fixed!

**Pull request merged**